internal/test/helpers.go

```diff
@@ -0,0 +1,116 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/DIMO-Network/shared/db"
+	"github.com/DIMO-Network/trips-api/internal/config"
+	"github.com/docker/go-connections/nat"
+	"github.com/pkg/errors"
+	"github.com/pressly/goose/v3"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const testDbName = "trips_api"
+
+// StartContainerDatabase starts postgres container with default test settings, and migrates the db. Caller must terminate container.
+func StartContainerDatabase(ctx context.Context, t *testing.T, migrationsDirRelPath string) db.Store {
+	settings := getTestDbSettings()
+	pgPort := "5432/tcp"
+	dbURL := func(_ string, port nat.Port) string {
+		return fmt.Sprintf("postgres://%s:%s@localhost:%s/%s?sslmode=disable", settings.DB.User, settings.DB.Password, port.Port(), settings.DB.Name)
+	}
+	cr := testcontainers.ContainerRequest{
+		Image:        "postgres:12.9-alpine",
+		Env:          map[string]string{"POSTGRES_USER": settings.DB.User, "POSTGRES_PASSWORD": settings.DB.Password, "POSTGRES_DB": settings.DB.Name},
+		ExposedPorts: []string{pgPort},
+		Cmd:          []string{"postgres", "-c", "fsync=off"},
+		WaitingFor:   wait.ForSQL(nat.Port(pgPort), "postgres", dbURL),
+	}
+
+	pgContainer, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: cr,
+		Started:          true,
+	})
+	if err != nil {
+		return handleContainerStartErr(ctx, err, t)
+	}
+	mappedPort, err := pgContainer.MappedPort(ctx, nat.Port(pgPort))
+	if err != nil {
+		return handleContainerStartErr(ctx, errors.Wrap(err, "failed to get container external port"), t)
```

Comment:
[github.com/pkg/errors](https://github.com/pkg/errors) has been archived for sometime now since error wrapping was added to the std lib.
```suggestion
		return handleContainerStartErr(ctx, fmt.Errorf("failed to get container external port: %w", err), t)
```